wurst/extensions/UnitExtensions.wurst

```diff
@@ -1,15 +1,15 @@
 package UnitExtensions
-import HashMap
+import StackNSplit
 import LinkedList
 import Lodash
 import GameConfig
 import PublicLibrary
-
+import Brambles
 
 public let BUFF_ENSNARE_GROUND = 'Beng'
 public let BUFF_ENSNARE_AIR = 'Bena'
 public let BUFF_ENTANGLE = 'BEer'
-
+public let BUFF_ENTANGLE_BRAMBLES = getBramblesEntangleBuffId()

```

Comment:
Generally we should prefer to do something like this:

```
public let BUFF_ENTANGLE_BRAMBLES = compiletime(getBramblesEntangleBuffId())
```

Reply:
I used a getter to give it a better public name since the brambles packages own variable name is DUMMY_ENTANGLE_BUFF and using that name would be misleading, eventually when we we get to the point where we have many of these buffs being added

Reply:
@Marsunpaisti maybe we can use a more explicit name in its own package--like `BUFF_ENTANGLE_BRAMBLES` like you've used here